contracts/VoucherKernel.sol

```diff
@@ -310,15 +310,20 @@ contract VoucherKernel is IVoucherKernel, Ownable, Pausable, UsingHelpers {
      * @param _tokenIdSupply   ID of the supply token (ERC-1155)
      * @param _issuer          Address of the token's issuer
      * @param _holder          Address of the recipient of the voucher (ERC-721)
+     * @param _paymentMethod   method being used for that particular order that needs to be fulfilled
      * @param _correlationId           ID of the current interaction with the smart contract for a specific user
      */
     function fillOrder(
         uint256 _tokenIdSupply,
         address _issuer,
         address _holder,
+        uint8 _paymentMethod,
```

Comment:
where is this payment method used?

Reply:
The payment method is a substantial part throughout the whole business logic.